simplified-app-ekirjasto/src/main/assets/txnative/fi/txstrings.json

```diff
@@ -790,10 +1057,10 @@
          "string" : "Käyttäjänimi"
       },
       "login_passkey" : {
-         "string" : ""
+         "string" : "Kirjaudu sisään pääsyavaimella"
       },
       "login_suomifi" : {
-         "string" : "Kirjaudu sisään Suomi.fi-tunnistuksella"
+         "string" : "Kirjaudu sisään suomi.fi:ssä"
```

Comment:
The original string (without this change) looks better to me? Should be fixed in Transifex (at least capitalized).